package.json

```diff
@@ -24,9 +24,11 @@
     "@eeacms/volto-widgets-view"
   ],
   "dependencies": {
-    "volto-slate": "*",
     "@eeacms/volto-widgets-view": "*"
```

Comment:
@silviubogan If we go for `peerDependencies` then I think we should move all Volto add-ons dependencies to `peerDependencies`

Reply:
@avoinea So I should do this change in all these PRs for all the addons that are now in `dependencies`?